docs/source/eventing/actions/label.rst

```diff
@@ -39,20 +42,33 @@ Optionally also set the background color and blinking.
    - HOUR
    - DAY
 
-.. versionadded:: 8.1.4.
+.. versionchanged:: nightly
 
 +----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
-| BLINK    | |no|             | |no|                | Blinking label text ``on`` or ``off``             |
+| DEVICE   | |yes|            | |yes|               | Device(s) to change                               |
```

Comment:
Can we make two tables? One with the options in the current release, and one with the options new to the nightly?

Reply:
Now there are two tables already. The first one shows the existing options in the current release and the second one shows all options for the nightly, so both the existing options ánd the new ones. 

Do you want me to change that second table to only show the new options and change ``versionchanged`` to ``versionadded``?

Reply:
> Do you want me to change that second table to only show the new options and change versionchanged to versionadded?

Yes